examples/show_detection_sample.py

```diff
@@ -0,0 +1,27 @@
+"""
+Load and show sample data
+=========================
+This example:
+- loads some sample data
+- adds the data to a napari viewer
+- loads the cellfinder-napari cell detection plugin
+- opens the napari viewer
+"""
+import napari
+
+from cellfinder_napari.sample_data import load_sample
+
+viewer = napari.Viewer()
+# Open plugin
+viewer.window.add_plugin_dock_widget(
+    plugin_name="cellfinder-napari", widget_name="Train network"
```

Comment:
```suggestion
    plugin_name="cellfinder-napari", widget_name="Cell detection"
```
To make it consistent with the docstring?